card_db_src/nl_nl/cards_nl_nl.json

```diff
@@ -121,6 +121,11 @@
         "extra": "De kaart die je pakt komt uit de voorraad en gaat naar je hand. Je mag geen _ Coin gebruiken om de kosten van de te pakken kaart te verhogen; de kaart kost altijd ten minste 0 Coin en ten hoogste 5 Coin. Na het pakken van de kaart leg je een kaart uit je hand op je trekstapel. Dat mag ook de juist gepakte kaart zijn.",
         "name": "Ambachtsman"
     },
+    "Astrolabe": {
+        "description": "Nu en aan het begin van je volgende beurt:<br><br>1 <*COIN*><br><br>+ 1 Aanschaf",
```

Comment:
@sumpfork This is the only description that uses "1 <*COIN*>". Should it be simply "+1 Coin"?

Reply:
Honestly, I'd like to change them all to be more markup-distinguishable rather than plain inline English words, but yeah, might as well be consistent for now and change it to "+1 Coin"

Reply:
Now that I look through the whole file there are other places with similar markup. Not sure whether to change those too now. Maybe better in another PR...

Reply:
yeah fair enough